app/javascript/tests/analytics/display/CustomizedAxisTick.test.js

```diff
@@ -8,6 +8,6 @@ const payload = { coordinate: 123, index: 4, offset: 32.1, value: 'some value' }
 describe('CustomizedAxisTick', () => {
   it('Properly renders all main components', () => {
     const wrapper = shallow(<CustomizedAxisTick x={0} y={0} payload={payload} />);
-    expect(wrapper.find(Text).exists()).toBeTruthy();
+    expect(wrapper.find(Text).exists()).toBe(true);
```

Comment:
I noticed there was a lot more `toBeTruthy` than `toBe(true)` before. Is there a reason to prefer the latter over the former?